.github/workflows/unit-tests.yml

```diff
@@ -0,0 +1,27 @@
+name: "Unit Tests"
+
+on: [push, pull_request]
+
+jobs:
+  unit-tests:
+    strategy:
+      fail-fast: true
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v2-beta
+        with:
+          node-version: '10'
+      - name: install yarn
```

Comment:
Do we need to separately install yarn. I used to think action/setup-node will do that too.

Reply:
it's how travis build was setup, to ensure the latest version of yarn